src/vm/ecalllist.h

```diff
@@ -958,8 +958,8 @@ FCFuncStart(gAppDomainFuncs)
 #ifndef FEATURE_CORECLR
     FCFuncElement("GetSecurityDescriptor", AppDomainNative::GetSecurityDescriptor)
     FCFuncElement("nIsDefaultAppDomainForEvidence", AppDomainNative::IsDefaultAppDomainForEvidence)
-    FCFuncElement("nGetAssemblies", AppDomainNative::GetAssemblies)
 #endif
+    FCFuncElement("nGetAssemblies", AppDomainNative::GetAssemblies)
```

Comment:
I am not sure whether the implementation works as expected with CoreCLR binder in all cases. You should check with @gkhanna79